tensorflow/compiler/mlir/xla/transforms/hlo_legalize_to_lhlo.cc

```diff
@@ -95,7 +89,7 @@ Value InsertAllocAndDealloc(Location loc, Value result,
 /// function to store that values held in the tensor.
 Value GetBufferForResultValue(Location loc, Value result,
                               ConversionPatternRewriter* rewriter) {
-  if (auto existing_memref = GetTensorStoreOrReturnMemRef(result)) {
+  if (auto existing_memref = GetTensorStore(result)) {
```

Comment:
It would be more consistent to replace TensorStore operations with copy and always allocate the result. Not necessarily in this PR but once we have a pass that optimizes redundant alloc/copy/dealloc operations away.

Reply:
Sure, I will address this in the next PR.